heartbeat/exportfs

```diff
@@ -417,6 +417,10 @@ testdir() {
 }
 exportfs_validate_all ()
 {
+	if ! echo "$OCF_RESKEY_fsid" | grep -q -E "^[A-Za-z0-9-]*$"; then
+		ocf_exit_reason "$OCF_RESKEY_fsid is not a valid fsid"
+		return $OCF_ERR_CONFIGURED
+	fi
```

Comment:
The regexp matching that description would be: '^(root|[[:xdigit:][:punct:]]*)$'. And then we'd need to hope that the description is precise. Somehow, I'd rather let this be as is, lest we introduce a regression.